components/script/dom/webgl2renderingcontext.rs

```diff
@@ -264,20 +337,256 @@ impl WebGL2RenderingContextMethods for WebGL2RenderingContext {
     }
 
     /// https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.5
```

Comment:
We should update these comments to point to the appropriate parts of the 2.0 spec instead.

Reply:
Ok, updated BindBuffer's comment. The BufferData and BufferSubData calls that come from the WebGL1 code are specified on the WebGL1 spec page.